nse_utility.h

```diff
@@ -8,9 +8,9 @@ class Target;
 #include "nmap_config.h"
 #endif
 
-#if HAVE_STDINT_H
+//#if HAVE_STDINT_H
```

Comment:
Don't change this; instead add this to `nmap_winconfig.h`:

```diff
diff --git a/nmap_winconfig.h b/nmap_winconfig.h
index bb59dfa..912d790 100644
--- a/nmap_winconfig.h
+++ b/nmap_winconfig.h
@@ -144,6 +144,8 @@
 
 #define HAVE_OPENSSL 1
 #define HAVE_SSL_SET_TLSEXT_HOST_NAME 1
+/* Since MSVC 2010, stdint.h is included as part of C99 compatibility */
+#define HAVE_STDINT_H 1
 
 #define LUA_INCLUDED 1
 #undef PCAP_INCLUDED
```